src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -198,6 +203,7 @@ def __init__(
         self.image_std = image_std if image_std is not None else OPENAI_CLIP_STD
         self.do_pad = do_pad
         self.do_center_crop = do_center_crop
+        self.crop_size = crop_size
```

Comment:
At the moment, this argument has no effect. It should be passed to the `self.center_crop` call in `preprocess` and default to `self.size` if unset here for backwards compatibility